gcc/config/arc/arc-c.c

```diff
@@ -106,3 +109,174 @@ arc_cpu_cpp_builtins (cpp_reader * pfile)
     builtin_define ("__big_endian__");
 
 }
+
+void
+arc_pr_jli_call_fixed (struct cpp_reader * pfile ATTRIBUTE_UNUSED)
+{
+  tree x;
+  enum cpp_ttype type;
+  char *symbol;
+  int index;
+
+  type = pragma_lex (&x);
+
+  if (type != CPP_OPEN_PAREN)
+  {
+    error ("expected '(' in #pragma jli_call_fixed");
+
```

Comment:
Please be more clear what you mean. For the most part it follows the horrible GCC style.